test_offline_mode/docker-compose.yaml

```
@@ -0,0 +1,64 @@
+volumes:
+  backup:
+
+networks:
+  internet:
+    driver: bridge
+  no-internet:
+    driver: bridge
+    internal: true
+
+services:
+  online-pdp:
+    container_name: online-pdp
+    build:
+      context: ..
+      dockerfile: Dockerfile
+    volumes:
+      - backup:/app/backup:rw
+    networks:
+      - internet
+    env_file: .env
+    healthcheck:
+      test: "wget --no-verbose --tries=1 --spider http://127.0.0.1:7000/healthy || exit 1"
+
+  offline-pdp:
+    container_name: offline-pdp
+    build:
+      context: ..
+      dockerfile: Dockerfile
+    volumes:
+      - backup:/app/backup:rw
+    networks:
+      - no-internet
+    env_file: .env
+    healthcheck:
+      test: "wget --no-verbose --tries=1 --spider http://127.0.0.1:7000/healthy || exit 1"
+    depends_on:
+      - online-pdp
```

change to this to prevent races
```suggestion
      online-pdp:
          condition: sevice_healthy
```